sources/packages/web/src/components/generic/Banner.vue

```diff
@@ -5,12 +5,9 @@
     </template>
     <v-row>
       <v-col cols="10"
-        ><div class="float-left label-value-normal">{{ summary }}</div></v-col
+        ><div class="label-value-normal">{{ summary }}</div></v-col
       >
-      <v-col cols="2"
-        ><div class="pl-8">
-          <slot name="actions"></slot></div
-      ></v-col>
+      <v-col cols="2"><slot name="actions"></slot></v-col>
```

Comment:
while using cols in a generic banner, please make sure to validate it's behavior in different resolutions is consistent for responsiveness. we can use lg,sm and md to handle them.